applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.hpp

```diff
@@ -0,0 +1,235 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Mohamed Nabi
+//
+
+#if !defined(KRATOS_GEO_THERMAL_ELEMENT_H_INCLUDED )
+#define  KRATOS_GEO_THERMAL_ELEMENT_H_INCLUDED
+
+// Project includes
+#include "includes/serializer.h"
+
+// Application includes
+#include "custom_utilities/element_utilities.hpp"
+#include "geo_mechanics_application_variables.h"
+
+namespace Kratos
+{
+
+    template< unsigned int TDim, unsigned int TNumNodes >
+    class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement :
+        public Element
+    {
+    public:
+
+        KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(TransientThermalElement);
+
+        typedef Element BaseType;
+
+        typedef std::size_t IndexType;
+        typedef Properties PropertiesType;
+        typedef Node <3> NodeType;
+        typedef Geometry<NodeType> GeometryType;
+        typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
+        typedef Vector VectorType;
+        typedef Matrix MatrixType;
+        typedef Element::DofsVectorType DofsVectorType;
+        typedef Element::EquationIdVectorType EquationIdVectorType;
+
+        /// The definition of the sizetype
+        typedef std::size_t SizeType;
+
+    	std::vector<ConstitutiveLaw::Pointer> mConstitutiveLawVector;
+        bool mIsInitialised = false;
+
+//        typedef typename BaseType::ElementVariables ElementVariables;
+
+        struct ElementVariables
+        {
+            ///Properties variables
+            double WaterDensity;
+            double SolidDensity;
+        	double Density;
```

Comment:
check the alignment.

Reply:
Done
Density is removed as it is never used